packages/arb-validator-core/gobridge/challenge.go

```diff
@@ -99,24 +95,39 @@ func (c *challenge) challengerResponded() {
 
 }
 
+func (c *challenge) resolveChallenge(winner common.Address, loser common.Address) {
+	rollup, ok := c.client.rollups[c.contractAddress]
```

Comment:
Isn't `c.contractAddress` the address of the challenge contract rather than the rollup contract?

Reply:
fixed